### ISSUE TITLE
"Price Strikethrough Text" and "SAVE $100" for on sale items not showing in Mobile Cart Page in Portrait Orientation"

### ISSUE THREAD

**Issue by calvinlchua** [woocommerce/woocommerce-gutenberg-products-block#2991]
## Describe the bug

In the "Cart" page, on-sale Items display both the the regular price and the sale price.  On Desktop, it shows the "Regular Price" with a strikethrough and a box that says "SAVE $100" as well as the new price which is working fine.

In mobile, it only shows the "Regular Price" and "Sale Price" side by side with no strikethrough and no box that says "SAVE $100"  It would be confusing for clients seeing two prices on the cart page.

It however, shows when the phone is rotated to landscape.  It just can't be seen in portrait

## To reproduce

1. Add an "On-sale" product to cart
2. Press "View Cart
## Expected behavior
The cart will display the product with its regular price with strikethrough formatting and the discount price next to it

## Screenshots
![IMG_8694](https://user-images.githubusercontent.com/69507569/89881463-222d7c00-dbf8-11ea-93e7-6860eb83b85e.jpg)
______________

![IMG_8696](https://user-images.githubusercontent.com/69507569/89881446-1d68c800-dbf8-11ea-95fe-ea16571c7aee.jpg)

### WordPress (please complete the following information):
* Core version: 
* Gutenberg version (if installed): 
* WooCommerce version: 4.3.2
* WooCommerce Blocks version: 3.1.0
* Site language:
* Any other plugins installed:
### Smartphone (please complete the following information):
* Device: iPhone XR
* OS: iOS 13.3
* Browser:

 Safari
* Version

**Comment by senadir** [comment]
Hello @calvinlchua, thank you for filling this issue, I can confirm that the sale tag is hidden on mobile, this is by design, to reduce clutter and redundant items on smaller screens, you can show it using this css code:
```css
.wc-block-components-sale-badge {
    display: inline-block !important;
}
```

However, we can ensure this class will stay the same always.

As for the price not being stricken, we will issue a fix for it.

**Comment by calvinlchua** [comment]
Hi Seghir,

Thank you for this.  May I ask if there is a way for me to fix the price
strikethrough issue for the meantime? or do I have to wait for the next
update of WooCommerce blocks?

**Comment by senadir** [comment]
@calvinlchua 

Yes, this can be done by adding this rule

```css
.wc-block-components-sale-badge {
	display: none;
}
```

**Comment by Aljullu** [comment]
Assuming the hidden sale badge is the expected behavior, this issue (the missing strike-through) has been fixed in #3065.
